server/routes/user.js

```diff
@@ -21,7 +21,22 @@ router.route('/:userId')
   /** DELETE /api/v1/user/:userId - Delete user */
   .delete(validate(paramValidation.byId), userCtrl.deleteUser);
 
+router.route('/:userId/device')
+  /** POST /api/v1/user/:userId/device - Add device to user */
+  .post(validate(paramValidation.addDevice), userCtrl.addDevice);
+
+router.route('/:userId/device/:deviceId')
```

Comment:
Would it make sense to do something like this https://github.com/expressjs/express/blob/4.13.1/examples/route-map/index.js#L52